.github/workflows/main.yml

```diff
@@ -26,6 +26,24 @@ jobs:
       - name: Test Docker Image
         run: docker/build-local.sh
 
+  error-prone-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Maven Install
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          ./mvnw install ${MAVEN_FAST_INSTALL} -pl '!presto-docs,!presto-server,!presto-server-rpm'
+      - name: Error Prone Checks
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          ./mvnw -B -e -T C1 test-compile -Dair.check.skip-all -P errorprone-compiler-presto -pl '
```

Comment:
I don't know how errorprone handles incremental compilation, so I'd prefer to have `clean compile test-compile` here. It should not incur any additional cost (we want errorprone to recompile everything anyway) and would give me more confidence.

Reply:
Nit: add `-Dair.check.skip-all` in .travis.yml too, for consistence.

Reply:
Done. Note that we don't need `compile` since `test-compile` is after that in the Maven lifecycle: https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Lifecycle_Reference